pkg/controller/acmeorders/sync_test.go

```diff
@@ -606,24 +609,32 @@ rUCGwbCUDI0mxadJ3Bz4WxR6fyNpBK2yAinWEsikxqEt
 }
 
 type testT struct {
-	order      *cmacme.Order
-	builder    *testpkg.Builder
-	acmeClient acmecl.Interface
-	expectErr  bool
+	order          *cmacme.Order
+	builder        *testpkg.Builder
+	acmeClient     acmecl.Interface
+	shouldSchedule bool
+	expectErr      bool
 }
 
 func runTest(t *testing.T, test testT) {
 	test.builder.T = t
 	test.builder.Init()
 	defer test.builder.Stop()
 
-	c := &controller{}
+	c := &controllerWrapper{}
 	c.Register(test.builder.Context)
 	c.accountRegistry = &accountstest.FakeRegistry{
 		GetClientFunc: func(_ string) (acmecl.Interface, error) {
 			return test.acmeClient, nil
 		},
 	}
+	gotScheduled := false
+	fakeScheduler := schedulertest.FakeScheduler{
+		AddFunc: func(obj interface{}, duration time.Duration) {
+			gotScheduled = true
+		},
+	}
+	c.scheduledWorkQueue = &fakeScheduler
```

Comment:
Huh! I didn't expect that to work. I knew you could call and get stuff directly from an embedded struct, but didn't realise that you could assign to an embedded struct without 
A bit confusing if you ask me.

Reply:
Yes, perhaps we should not use the wrapper. I added it for consistency with the certificates controllers.

I think though that `c.controller.scheduledWorkQueue = &fakeSchedule` would be worse as we do i.e `c.Sync` where `c` is the controller.